src/css-grids/cssgrids.css

```diff
@@ -0,0 +1,48 @@
+.main-container {
+    background: url("flag.jpg");
+    height: 1000px;
+    width: 1000px;
```

Comment:
change height and width to 100%. no need to give it a static h&w. if you did want it to stop growing at a certain size you can do 
`height: 100%;
max-height: 1000px;'
width: 100%;
max-width: 1000px;`